my-code/frontend/src/components/MovieCard/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Container, Overlay } from "./styles";
+import { useHistory } from "react-router";
+import useLike from "../../hooks/useLike";
+interface IProps {
+  Poster: string;
+  Title: string;
+  Type: string;
+  Year: string;
+  imdbID: string;
+}
+
+const MovieCard: React.FC<IProps> = ({ Poster, Title, Year, imdbID }) => {
+  const history = useHistory();
+  const { liked, handleLike } = useLike(imdbID);
+
+  const handleRoute = () => {
+    history.push(`/m/${imdbID}`);
+  };
+
+  return (
+    <Container
+      onClick={handleRoute}
```

Comment:
this doesn't work with cmd + click to open in new tab.
also a11y could be improved. this action is a navigation so you should use an anchor or at least provide the necessary attributes to the div